docs/roles/system_manager/manage_users.md

```diff
@@ -82,9 +86,9 @@ Users that didn't have a `GroupName` specified in the `user_details_template.csv
 - Open a `Powershell` command window with elevated privileges
 - Run `C:\Installation\Run_ADSync.ps1`
 
-### {{iphone}} Changing a users phone number
+### {{iphone}} Edit user details
 
-If a user has provided an incorrect phone number, or has a new phone number, the phone number associated with their account will need to be updated.
+The `DC1` is the source of truth for user details. If these details need to be changed, the user should be deleted in the Azure Portal and re-created from the `DC1`.
```

Comment:
Will delegate to @jemrobinson here.

I _think_ user details should sync between AAD and the local AD, so deleting and recreating the user shouldn't often be necessary. I recall one entry being fixed, which it seems was email address and not phone number. That might be the only case where you would make a new user.

Reply:
Yes, the users in AzureAD are synchronised from the local AD on the DC. So all changes (apart from resetting MFA) should be done on the local DC.

Reply:
An that is unidirectional? _I.e._ no sync from AAD to local AD?

Reply:
There is password hash synchronisation, but I think that's just that when a password is reset the hash is sent to both AzureAD and the local AD, I don't think the password hash is synchronised from AzureAD to the local AD.

The only thing I can think of that's synchronised from AzureAD to the local AD is the AzureAD anchor attribute which identifies which local AD user maps to which AzureAD user.